src/airtable/proposals/proposal_standings.js

```diff
@@ -232,7 +232,8 @@ const getProposalRecord = async (proposal, allProposals) => {
       'Proposal Standing': newStanding,
       'Disputed Status': disputed,
       'Outstanding Proposals': undefined,
-      'Reason Rejected': validProposal === true ? undefined : validProposal
+      'Reason Rejected':
+        validProposal === true ? 'Valid Proposal' : validProposal
```

Comment:
Why are we setting the 'Reason Rejected' to 'Valid Proposal'? It is a little bit confusing, undefined makes more sense for me.